test/test_binomial.cpp

```diff
@@ -716,6 +716,18 @@ void test_spots(RealType T)
 
    check_out_of_range<boost::math::binomial_distribution<RealType> >(1, 1); // (All) valid constructor parameter values.
 
+   // TODO: Generic ibeta_power_terms has accuracy issue when long
+   // double is not precise enough, causing overflow in this case.
+   if(!std::is_same<RealType, real_concept>::value ||
```

Comment:
What happens here when T is a double, does that still work?  Ideally we shouldn't be having spurious internal overflow as a) it breaks types with no infinity (this is potentially true for long double too) and b) it sets the hardware overflow flag which can trip up downstream consumers.

Reply:
The problem is in beta.hpp#478 onwards.  Don't see a trivial fix just yet :(

inv_distrete_quantile.hpp#150 also needs adjusting to `else if((adder != 0) && (a + adder != a))` to speed up this new test case.

Reply:
With built-in `double` and `long double` it works fine because Lanczos approximation is enabled.